.github/actions/test-alpine/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Test
+inputs:
+  runTestsParameters:
+    default: ''
+    required: false
+  jitType:
+    default: 'disable'
+    required: false
+runs:
+  using: composite
+  steps:
+    - shell: bash
+      run: |
+        set -x
+        export SKIP_IO_CAPTURE_TESTS=1
+        export STACK_LIMIT_DEFAULTS_CHECK=1
+        sapi/cli/php run-tests.php -P -q ${{ inputs.runTestsParameters }} \
+          -d opcache.jit=${{ inputs.jitType }} \
+          -d opcache.jit_buffer_size=64M \
+          -j$(($(nproc) - 1)) \
```

Comment:
You can probably do `-j$(nproc)`. The `-1` was used as an attempt to fix crashes specific to macOS.